packages/react-notifications/src/components/notifications-list/notifications-list.tsx

```diff
@@ -0,0 +1,273 @@
+import Notification from '../notification';
+import GenericNotification from '../notification-kinds/generic';
+import ApiErrorNotification from '../notification-kinds/api-error';
+import UnexpectedErrorNotification from '../notification-kinds/unexpected-error';
+import { useCustomNotificationComponent } from '../map-notification-to-component';
+import {
+  selectGlobalNotifications,
+  selectPageNotifications,
+  selectSideNotifications,
+} from './selectors';
+import { TAppState } from './types';
+import { reportErrorToSentry } from '@commercetools-frontend/sentry';
+
+type Props = {
+  domain: TAppNotificationDomain;
+};
+
+const getStyles = (props: Props) => {
+  const baseStyles = css`
+    color: ${customProperties.colorSurface};
+    position: relative;
+    width: 100%;
+    z-index: 10000;
+  `;
+
+  switch (props.domain) {
+    case NOTIFICATION_DOMAINS.GLOBAL:
+      return css`
+        ${baseStyles};
+        text-align: center;
+        width: 100% !important;
+
+        > {
+          display: inline-flex;
+          padding-right: 48px;
+          height: 100%;
+          width: 100%;
+        }
+      `;
+    case NOTIFICATION_DOMAINS.PAGE:
+      return css`
+        ${baseStyles};
+        padding: ${customProperties.spacingM} ${customProperties.spacingL};
+      `;
+    case NOTIFICATION_DOMAINS.SIDE:
+      return css`
+        ${baseStyles};
+        position: absolute;
+        text-align: left;
+        height: 0;
+        overflow: visible;
+      `;
+    default:
+      return css``;
+  }
+};
+
+const NotificationsList = (props: Props) => {
+  const dispatch = useDispatch();
+  const mapCustomNotificationToComponent = useCustomNotificationComponent();
+
+  switch (props.domain) {
+    case NOTIFICATION_DOMAINS.GLOBAL: {
+      const notifications = useSelector<TAppState, TAppNotificationGlobal[]>(
+        selectGlobalNotifications
+      );
+      return (
+        <div css={getStyles(props)}>
+          {notifications.map(notification => {
+            // 1. Check if there is a custom notification component first
+            const CustomNotificationComponent = mapCustomNotificationToComponent(
+              notification
+            ) as Notification;
+            if (CustomNotificationComponent) {
+              return (
+                <CustomNotificationComponent
+                  key={notification.id}
+                  notification={notification}
+                  dismiss={() => {
+                    dispatch(removeNotification(notification.id));
+                  }}
+                />
+              );
+            }
+
+            switch (notification.kind) {
+              case NOTIFICATION_KINDS_GLOBAL.error:
+              case NOTIFICATION_KINDS_GLOBAL.warning:
+              case NOTIFICATION_KINDS_GLOBAL.info:
+              case NOTIFICATION_KINDS_GLOBAL.success: {
+                const { values, ...genericNotification } = notification;
```

Comment:
The `GenericNotification` does not accept the `values` prop, so we need to "remove" it.